tests/Aspire.Azure.AI.OpenAI.Tests/AzureOpenAISettingsTests.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace Aspire.Azure.AI.OpenAI.Tests;
+
+public class AzureOpenAISettingsTests
+{
+    [Fact]
+    public void TracingIsEnabledWhenAzureSwitchIsSet()
```

Comment:
I don't think we need these tests for this component. They are necessary for ServiceBus because ServiceBus doesn't enable tracing by default (where all of the HTTP SDKs do).